extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/KafkaIndexTaskTest.java

```diff
@@ -697,6 +697,23 @@ public void testIncrementalHandOffMaxTotalRows() throws Exception
     }
     final Map<Integer, Long> nextOffsets = ImmutableMap.copyOf(task.getRunner().getCurrentOffsets());
 
+      // Check metrics
+    Assert.assertEquals(8, task.getRunner().getRowIngestionMeters().getProcessed());
```

Comment:
It looks like maybe some logic got shuffled around, could this be related to failing tests?

Reply:
yeah looks like they got moved earlier in the code. I have repositioned them and tests as passing on my local. Hopefully, this should build fine. :)